docs/self-managed/identity/deployment/configuration-variables.md

```diff
@@ -134,14 +135,93 @@ steps:
+| `IDENTITY_TENANTS_0_MEMBERS_0_APPLICATIONID` | If adding an **application** as a member of the tenant, the application ID. If additional members have been added to the tenant, ensure the second `0` value matches that of its type (for example, `IDENTITY_TENANTS_0_MEMBERS_1_APPLICATIONID`). | `zeebe`       |
+| `IDENTITY_TENANTS_0_MEMBERS_0_USERNAME`      | If adding a **user** as a member of the tenant, the username. If additional members have been added to the tenant, ensure the second `0` value matches that of its type (for example, `IDENTITY_TENANTS_0_MEMBERS_1_USERNAME`).                    | `user-name`   |
+| `IDENTITY_TENANTS_0_MEMBERS_0_GROUPNAME`     | If adding a **group** as a member of the tenant, the name of the group. If additional members have been added to the tenant, ensure the second `0` value matches that of its type (for example, `IDENTITY_TENANTS_0_MEMBERS_1_GROUPNAME`).         | `group-name`  |
+
+:::note
+Each property will only apply if `TYPE` is set accordingly. For example, `IDENTITY_TENANTS_0_MEMBERS_0_USERNAME` will only function if `TYPE` is set to `USER`.
+:::
+
+When converting the above variables for use in the Helm chart, ensure multiple tenants or members are added as a list. For example, a valid `application.yaml` configuration might look like the following:
+
+```
+identity:
+   tenants:
+    - name: Tenant Name
+      tenantId: tenant-name
+      members:
+        - type: GROUP
+          group-name: "Test Group"
+        - type: USER
+          username: test-user
+        - type: APPLICATION
+          application-id: test-application
+```
+
+## Configure applications
+
+The following variables can be used to configure [applications](/self-managed/identity/user-guide/additional-features/incorporate-applications.md). To add additional applications, increment the `0` value for each variable (for example, `KEYCLOAK_CLIENTS_1_NAME`).
+
+| Environment variable                                  | Description                                                           | Default value            |
+| ----------------------------------------------------- | --------------------------------------------------------------------- | ------------------------ |
+| `KEYCLOAK_CLIENTS_0_NAME`                             | A human-readable name for the client.                                 | zeebe                    |
+| `KEYCLOAK_CLIENTS_0_ID`                               | The client ID.                                                        | `${ZEEBE_CLIENT_ID}`     |
+| `KEYCLOAK_CLIENTS_0_SECRET`                           | The client secret.                                                    | `${ZEEBE_CLIENT_SECRET}` |
+| `KEYCLOAK_CLIENTS_0_TYPE`                             | The type of client, one of `CONFIDENTIAL`, `PUBLIC`, or `M2M`.        | `M2M`                    |
```

Comment:
@ThorbenLindhauer 

> If the type value is not M2M, then the variable KEYCLOAK_CLIENTS_0_ROOTURL must be provided too (otherwise Identity won't start). It's not fully clear to me what the purpose is and what a proper example value would be, but I can help figuring this out when we get to it.

Have you been able to track down the purpose and an example?